lib/graphql/schema.ts

```diff
@@ -1385,13 +1383,14 @@ export type InvoiceSettingsOutput = {
   country?: Maybe<Scalars['String']>;
   email?: Maybe<Scalars['String']>;
   phoneNumber?: Maybe<Scalars['String']>;
-  /** If a user's setting has a logoPath, we calculate a url to the thumbnail from it */
-  logoUrl?: Maybe<Scalars['String']>;
   /** Number of days which get added to today's date to create a default value for due date on invoice creation form */
   dueDateDefaultOffset?: Maybe<Scalars['Float']>;
   nextInvoiceNumber?: Maybe<Scalars['Float']>;
   taxNumber?: Maybe<Scalars['String']>;
   vatNumber?: Maybe<Scalars['String']>;
+  id?: Maybe<Scalars['String']>;
```

Comment:
🤔  I set the `id` as mandatory in the[ graphql type file](https://github.com/kontist/backend/pull/9796/files#diff-227fe4028ba522a62e6312b28ec45e2f2e92bfb0d093a282d68c0c183d9a77d8R50), so I'm wondering why the id is set as optional here.

Reply:
beucase of https://github.com/kontist/backend/pull/9796/files#diff-227fe4028ba522a62e6312b28ec45e2f2e92bfb0d093a282d68c0c183d9a77d8R49 :)

Reply:
thank you, schema is correct now :)